### DISCUSSION TITLE
How to disable certain completion groups like "ancestor directory"?

### DISCUSSION THREAD

**Discussion by chrisgrieser** [marlonrichert/zsh-autocomplete#659] [Q&A] [answered]
Thank you so much for this plugin!

So I am a bit of a noob when it comes to zsh completions. Like, how would I go about disabling a certain group like the ancestor directories? I basically never use it, and getting them suggested feels unnecessarily noisy.

**Comment by marlonrichert** [comment, marked as answer]
That's currently not possible, but I agree that the feature is not particularly useful. It was once requested and easy to implement, but I think I'll just remove it at this point: #675.
